sw/host/opentitanlib/src/rom_patch/mod.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO(lowRISC/opentitan#15443): Fix this lint.
+#[allow(clippy::module_inception)]
```

Comment:
I think you can remove these lines - the parent module is named `rom_patch` and the child is just `patch`, so there shouldn't be a warning.

Also, nit: do you need this `patch` submodule? Are you planning to add more modules to `rom_patch` in the future?

Reply:
I removed that comment. And I also removed `patch.rs` to keep all the code under `mod.rs`.